app/models/filters/filter_base.rb

```diff
@@ -1522,6 +1522,8 @@ def organization_names
     end
 
     def project_names(ids = project_ids)
+      return [] if ids.blank?
```

Comment:
short circuit since if ids are empty, we won't return anything. Avoids unnecessarily calling project_name